### PR TITLE
Run Chrome headless tests on CI

### LINKED ISSUE #622: web-sys: Add Chrome headless testing to CI
We'll want to edit the travis config to download `chromedriver` and also probably Chrome itself. We should the configure js-sys and web-sys tests to both run with `geckodriver` and `chromedriver` on CI, ensuring that tests pass in both Firefox and Chrome.

### PR DESCRIPTION
**Pull request by alexcrichton**
Closes #622

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -78,10 +82,14 @@ matrix:
       install:
         - *INSTALL_NODE_VIA_NVM
         - *INSTALL_GECKODRIVER
+        - *INSTALL_CHROMEDRIVER
       script:
-        - cargo test -p web-sys --target wasm32-unknown-unknown
+        - export RUST_LOG=wasm_bindgen_test_runner
+        - CHROMEDRIVER=`pwd`/chromedriver cargo test -p web-sys --target wasm32-unknown-unknown
```
Comment (fitzgen) [inline .travis.yml]: I think this should probably be a separate travis job

File: .travis.yml [hunk 2]
```diff
@@ -90,10 +98,14 @@ matrix:
       install:
         - *INSTALL_NODE_VIA_NVM
         - *INSTALL_GECKODRIVER
+        - *INSTALL_CHROMEDRIVER
       script:
-        - cargo test -p js-sys --target wasm32-unknown-unknown
+        - export RUST_LOG=wasm_bindgen_test_runner
+        - CHROMEDRIVER=`pwd`/chromedriver cargo test -p js-sys --target wasm32-unknown-unknown
```
Comment (fitzgen) [inline .travis.yml]: ditto

**Comment by alexcrichton** [comment]
Ok I think I figured out the incantation to get this working.

@fitzgen I agree in principle yeah but CI already takes so long I don't think we can spare two more whole builders when running Chrome after Firefox takes ~seconds

**Comment by fitzgen** [comment]
> @fitzgen I agree in principle yeah but CI already takes so long I don't think we can spare two more whole builders when running Chrome after Firefox takes ~seconds

Ah, OK. I was thinking this would help speed things up, but if not then definitely *don't* do this!

**Pull request merged**